apps/i18n/common/pl_pl.json

```diff
@@ -1006,6 +1024,34 @@
   "parent": "Rodzic",
   "parents": "Rodzice",
   "parentsAndStudents": "Rodzice i uczniowie",
+  "parentLetterClever1": "Poproś uczniów, aby zalogowali się na swoje konto Clever na stronie [www.clever.com] ({cleverLink}) (kliknij „Zaloguj się jako student” w prawym górnym rogu)",
```

Comment:
@castro-jorge a lot of these translations break Markdown syntax

Reply:
Thanks. Getting in touch with the translator.